frb_codegen/src/utils.rs

```diff
@@ -71,6 +71,14 @@ pub fn get_symbols_if_no_duplicates(configs: &[crate::Opts]) -> Result<Vec<Strin
     Ok(all_symbols)
 }
 
+pub fn first<T>(vec: &[T]) -> Option<&T> {
+    vec.get(0)
+}
+
+pub fn remove_first<T>(vec: &mut Vec<T>) -> T {
```

Comment:
uber-nit: `remove(0)` sounds already quite short (even shorter if count literally)

It is the `.single()` (if you need it) that is more nontrivial, because that includes a length check etc

Reply:
I don't know where `.single` is

Reply:
If you do not need it then no need to write down `.single` :) Forget whether it is this PR or another PR that needs it